api_testing/support/helpers.js

```diff
@@ -0,0 +1,29 @@
+export const getRandomLetters = (length) => {
+  let result = ''
+  const alphabet = 'abcdefghijklmnopqrstuvwxyz'
+  for (let i = 0; i < length; i++) {
+    result += alphabet.charAt(Math.floor(Math.random() * alphabet.length))
+  }
+  return result
+}
+
+export function generateRegisterUserData() {
+  const randomLetters = getRandomLetters(10)
+  return {
+    email: `user_${randomLetters}@mymail.com`,
+    password: 'Yauhen12345',
+    username: `user_${randomLetters}`,
+    bio: '',
+    image: '',
+  }
```

Comment:
better put required fields that do not have real test value into fixtures, to avoid hardcoding them every time, and making maintenance easier:

fixtures/user.json:
```
{
    "email": "",
    "password": "",
    "username": "",
    "bio": "",
    "image": ""
}
```

```suggestion
  import defaultUser from '../fixtures/user.json';

  const username = `user_${getRandomLetters(10)}`
  return {
    ...defaultUser,
    email: `${username}@mymail.com`,
    password: 'Yauhen12345',
    username,
  }
```

Reply:
Done. Also did the same for article, just for consistency.